src/app/src/reducers/ClientInfoReducer.js

```diff
@@ -0,0 +1,21 @@
+import { createReducer } from 'redux-act';
+import update from 'immutability-helper';
+
+import {
+    startFetchClientInfo,
+    failFetchClientInfo,
+    completeFetchClientInfo,
+} from '../actions/clientInfo';
+
+const initialState = Object.freeze({ fetched: false, countryCode: null });
+
+export default createReducer({
+    [startFetchClientInfo]: () => initialState,
+    [failFetchClientInfo]: (state, payload) => update(state, {
+        fetched: { $set: true }
+    }),
+    [completeFetchClientInfo]: (state, payload) => update(state, {
+        fetched: { $set: true },
+        countryCode: { $set: payload.country.code }
+    }),
```

Comment:
Would there ever be a need to clear this during the running of the app? It seems like the answer is no but I thought I'd check.

Reply:
I don't think we would ever need to clear the fetched client info.